clang/test/CodeGen/builtins-nvptx-native-half-type-native.c

```diff
@@ -0,0 +1,117 @@
+// REQUIRES: nvptx-registered-target
+//
+// RUN: %clang_cc1 -ffp-contract=off -triple nvptx-unknown-unknown -target-cpu \
+// RUN:   sm_86 -target-feature +ptx72 -fcuda-is-device -x cuda -emit-llvm -o - %s \
```

Comment:
I assume you don't need the target-feature or target-cpu

Reply:
They're probably required since the NVPTX intrinsics have requirements on sm and ptx versions.